app/pulp/app/serializers/catalog.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from django.core import validators
+
+from pulp.app import models
+from pulp.app.serializers import ModelSerializer
+
+
+class DownloadCatalogSerializer(ModelSerializer):
+    url = serializers.CharField(
+        help_text="The URL used to download the related artifact.",
+        allow_blank=True, required=False, validators=[validators.URLValidator],
```

Comment:
Is this something we want API users to be able to create? I would have expected no validators and `read_only=True`, but I don't remember how these get created so this might be an unhelpful note.